src/System.ServiceModel.Primitives/tests/Description/ContractDescriptionTest.cs

```diff
@@ -202,4 +202,23 @@ public static void ContractDescription_GetContract()
         Assert.Equal(typeof(IDescriptionTestsService).Name, contractDescription.ContractType.Name);
         Assert.Equal("http://tempuri.org/", contractDescription.Namespace);
     }
+
+    [WcfFact]
+    public static void OperationDescription_BeginEndSyncMethod_Property()
+    {
+        ContractDescription contractDescription = ContractDescription.GetContract(typeof(IDescriptionTestsServiceBeginEndGenerated));
+        Assert.Equal(2, contractDescription.Operations.Count);
+        foreach(OperationDescription operation in contractDescription.Operations)
+        {
+            Assert.NotNull(operation.BeginMethod);
+            Assert.NotNull(operation.EndMethod);
```

Comment:
Can you also do `typeof(IDescriptionTestsServiceBeginEndGenerated).GetMethod(nameof(IDescriptionTestsServiceBeginEndGenerated.BeginMessageRequestReply)` and do an equals check. Do the same for End method and the sync method.

Reply:
Thanks for review! Equal check added.